.drone.yml

```diff
@@ -114,15 +114,17 @@ build:
       - export WORKER1=$VM2
       - export WORKER2=`$GOVC_GET_IP Ubuntu.14.04`
       - #
-      - ./misc/drone-scripts/deploy-and-test-wrapper.sh deployplugin $GOVC_URL_6_5 $VM1 $VM2 $$BUILD_NUMBER < /dev/null
+      - export CI_NODE4=`$GOVC_GET_IP photon.vsan`
+      - #
+      - ./misc/drone-scripts/deploy-and-test-wrapper.sh pluginSanityCheck $GOVC_URL_6_5 $VM1 $VM2 $$BUILD_NUMBER installbuilds < /dev/null
       - #
       - echo "=> Running tests on ESX_6.5" `date` > /dev/null
       - #
       - ./misc/drone-scripts/deploy-and-test-wrapper.sh runtests $GOVC_URL_6_5 $VM1 $VM2 $$BUILD_NUMBER < /dev/null
       - #
       - echo "=> Running tests on VSAN ESX_6.5" `date` > /dev/null
       - #
-      - export VM1=`$GOVC_GET_IP photon.vsan`
+      - export VM1=$CI_NODE4
       - export VM2=`$GOVC_GET_IP Ubuntu.14.04`
```

Comment:
Since we are using variable name in above line, let's also use variable name (i.e. $WORKER2) in this line.